app/jobs/purge_soft_deleted_records_job.rb

```diff
@@ -0,0 +1,112 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# hard-delete HMIS old soft-deleted records
```

Comment:
I don't think "HMIS" is a clear descriptor here, since this isn't focused on "HMIS" as in the data source / app, nor is it limited to only HUD-HMIS-spec records (since we include custom tables).

Maybe we could just say it cleans up "source data". In addition to cleaning up "destination Clients"? Thats seems more in line with the language we use elsewhere, but I could be wrong.

Reply:
good point, thanks. That comment is stale. I've improved the documentation of the class and params